Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:18-alpine3.17
+
+WORKDIR /usr/src/app
+COPY package*.json ./
+COPY tsconfig.json ./
+COPY tslint.json ./
+COPY yarn.lock ./
+
+COPY src ./src
+COPY types ./types
+COPY abi ./abi
```

Comment:
It's not necessary to put all here, just the build output like here
https://github.com/Giveth/giveconomy-balance-aggregator/blob/main/Dockerfile
And then run the service by running the js file with node

Reply:
@aminlatifi Please share the full build and run commands in the docs so that I can know how to build the Dockerfile. Any ENV I should be aware of?? Also is this running at any port?? 3000?? If so we should also change the compose file